astropy/units/quantity.py

```diff
@@ -114,6 +115,16 @@ def __next__(self):
     next = __next__
 
 
+class QuantityColumnInfo(ColumnInfo):
+    _attrs_from_parent = set(['dtype', 'unit'])
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Ah yes, it's a leftover from when `__init__` here was doing something.  I just blindly deleted that one line.